juniper_official/Kernel/detect-routing-socket-errors.rule

```diff
@@ -0,0 +1,124 @@
+/*
+ * This rule checks for the total number of routing socket errors at any given time in the system.
+ * Routing socket errors includes errors returned from rtsock, veto and mproc layers.
+ * It monitors the total number of routing socket errors and raises an alarm if there are too many of 
+ * these detected for significant duration of time. 
+ * 
+ * This rule will check if the error has been increased by 100 from its previous value in ALL the samples 
+ * for the last 70 or 130 seconds and raises the alarm accordingly.
+ */
+iceberg {
```

Comment:
Please address all above comments i.e. related field type, removal of periods, iceberg to healthbot, sensor frequency, trigger frequency, etc. in all rules